src/ResourceManager/Network/Commands.Network.Test/ScenarioTests/NetworkWatcherAPITests.ps1

```diff
@@ -402,7 +372,7 @@ function Test-PacketCapture
     # Setup
     $resourceGroupName = Get-ResourceGroupName
     $nwName = Get-ResourceName
-    $location = "eastus"
+    $location = "centraluseuap"
```

Comment:
can we use prod regions only in testing?

Reply:
No, we can't. There was a bug in APIs. Fix will be rolled out to prod regions in 2 weeks. Right now fix is deployed only in centraluseuap. So, if you want tests to pass reliably, they should be run in centralueeuap